json-transport/tests/bench.rs

```diff
@@ -0,0 +1,108 @@
+// Copyright 2018 Google LLC
```

Comment:
Rather than duplicating these tests between all transport crates -- which makes it harder to maintain -- what if we had a `transport-tests` crate that depended on the other transports maintained directly in tarpc (json, bincode, ...)? Then we could convert the body of these tests to library functions that could be reused in multiple tests.